### PR TITLE
Explain SHOULD NOT in section 4

### PR DESCRIPTION
**Pull request by tfpauly**

### CODE REVIEW

File: draft-ietf-ohai-svcb-config.md [hunk 1]
```diff
@@ -131,9 +131,11 @@ The "ohttp" parameter can be included in the mandatory parameter
 list to ensure that clients that do not support access via Oblivious HTTP
 do not try to use the service. Services that mark the "ohttp"
 parameter as mandatory can, therefore, indicate that the service might
-not be accessible except through Oblivious HTTP. Services that optionally
-support access using Oblivious HTTP SHOULD NOT mark the "ohttp" parameter
-as mandatory. Note that since multiple SVCB responses can be provided for
+not be accessible except through Oblivious HTTP. Services that
+support access both with and without Oblivious HTTP SHOULD NOT mark
```
Comment (tfpauly) [inline draft-ietf-ohai-svcb-config.md]:
```suggestion
support access both with and without Oblivious HTTP SHOULD NOT mark
```

**Review by martinthomson — COMMENTED** [review]
And here I was, angling for less text.

> Services can mark the "ohttp" parameter as mandatory if the service is only accessible using Oblivious HTTP.  Marking the "ohttp" parameter as mandatory will cause clients that do not understand the parameter to ignore that service endpoint [terminology?]. The presence of the "ohttp" parameter without a mandatory indicator advertises a service that is optionally available using Oblivious HTTP.  Note also that multiple SVCB records can be provided to indicate separate services with different configurations.

File: draft-ietf-ohai-svcb-config.md (hunk 1 above)
Comment (tireddy2) [inline draft-ietf-ohai-svcb-config.md]: Any specific reason not to use "MUST NOT" ?
I can't think of a valid exception why "ohttp" parameter would be set to mandatory when the service is accessible with and without Oblivious HTTP.

**Comment by tfpauly** [comment]
Updated the text based on @martinthomson 's suggestion, making this non-normative.

**Review by tireddy2 — APPROVED** [review]

**Pull request merged**